<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PSO - Iago Alves</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Rodar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_times">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Partículas</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_part">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Dimensões</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_dim">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Loops</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_loop">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Topologia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_top">
        <item>
         <property name="text">
          <string>Estrela (global)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Anel (local)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mestre Escravo (focal)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Função</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_alg">
        <item>
         <property name="text">
          <string>Esfera</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rotated Rastrigin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rosenbrock</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Velocidade</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_vel">
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Inercial</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Inercial decaindo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Clerc</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Confiabilidade:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="lb_conf">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Tempo de execução</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_time">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>82</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bt_triple">
        <property name="text">
         <string>Executar 3 funçoes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_work">
        <property name="text">
         <string>Iniciar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QCustomPlot" name="plotter" native="true">
      <property name="minimumSize">
       <size>
        <width>670</width>
        <height>290</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
